scripts/myMangaDex.js

```diff
@@ -1493,14 +1535,7 @@ class MyMangaDex {
 		// Informations
 		await this.getTitleInfos();
 		await this.fetchMyAnimeList();
-		// Save LocalStorage to MAL values if it's higher
-		if (this.manga.lastMyAnimeListChapter > this.manga.lastMangaDexChapter) {
-			this.manga.lastMangaDexChapter = this.manga.lastMyAnimeListChapter;
-			if (this.options.saveAllOpened) {
-				this.insertChapter(this.manga.lastMangaDexChapter);
-			}
-			await updateLocalStorage(this.manga, this.options);
-		}
+
```

Comment:
```suggestion
		// Save LocalStorage to MAL values if it's higher
		if (this.manga.lastMyAnimeListChapter > this.manga.lastMangaDexChapter) {
			this.manga.lastMangaDexChapter = this.manga.lastMyAnimeListChapter;
			if (this.options.saveAllOpened) {
				this.insertChapter(this.manga.lastMangaDexChapter);
			}
			await updateLocalStorage(this.manga, this.options);
		}
```
I forgot to add this to the new reader and there were no problems, this block should not be needed